### ISSUE TITLE
 training loss

### ISSUE THREAD

**Issue by favor6** [Daniil-Osokin/lightweight-human-pose-estimation.pytorch#222]
hi，What is the loss function of your training？

**Comment by favor6** [comment]
Hi，why did I train the 200000 model and the AP value was 0.037 during verification? What's the problem?

**Comment by Daniil-Osokin** [comment]
Hi! This looks strange. Did you follow steps 1-3 from the [training guide](https://github.com/Daniil-Osokin/lightweight-human-pose-estimation.pytorch#training)?

**Comment by favor6** [comment]
Yes, I had this problem in the first step of training.
<img width="418" alt="100000" src="https://user-images.githubusercontent.com/90312746/144349394-69259929-d11a-42b6-a215-86d1206525e7.PNG">
This is the verification result of my training 100000. I don't know why this result occurs.

**Comment by Daniil-Osokin** [comment]
It is strange. Can you please show the beginning of training, i.e. parameters to run the `train.py` script?

**Comment by favor6** [comment]
Thank you. I have solved it. There is a mistake in one of the parameters in the training.

**Comment by Daniil-Osokin** [comment]
Ok, closing the issue.
